From eeb68f0a160dddd1e841b19a76d604bfe5569b9d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 28 Nov 2014 15:32:17 +0100
Subject: [PATCH] fips: add libfreeblpriv3.so and libfreeblpriv3.chk

(cherry picked from commit 3659d64df3fe87e2ec2ca6a55c8f456dabb953db)
---
 modules.d/01fips/module-setup.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules.d/01fips/module-setup.sh b/modules.d/01fips/module-setup.sh
index 35cfdc5a..2b0e836c 100755
--- a/modules.d/01fips/module-setup.sh
+++ b/modules.d/01fips/module-setup.sh
@@ -38,7 +38,8 @@ install() {
 
     inst_libdir_file libsoftokn3.so libsoftokn3.so \
         libsoftokn3.chk libfreebl3.so libfreebl3.chk \
-        libssl.so 'hmaccalc/sha512hmac.hmac' libssl.so.10
+        libssl.so 'hmaccalc/sha512hmac.hmac' libssl.so.10 \
+        libfreeblpriv3.so and libfreeblpriv3.chk
 
     inst_multiple -o prelink
     inst_simple /etc/system-fips
